### PR TITLE
Move docs to documentation repository

### PR DESCRIPTION
**Pull request by coot**

### CODE REVIEW

File: io-sim-classes/src/Control/Monad/Class/MonadSTM/Strict.hs [hunk 1]
```diff
@@ -50,6 +51,10 @@ data StrictTVar m a = StrictTVar
    , tvar      :: !(Lazy.LazyTVar m a)
    }
 
+-- TODO: it can be removed after 'Server.Socket' is re-written using 'MonadSTM'
```
Comment (mrBliss) [inline io-sim-classes/src/Control/Monad/Class/MonadSTM/Strict.hs]: This change is unrelated to the PR

**Comment by rvl** [comment]
https://github.com/input-output-hk/ouroboros-network-docs
